src/shogun/clustering/KMeansBase.h

```diff
@@ -129,6 +154,9 @@ class KMeansBase : public RandomMixin<DistanceMachine>
 		/** Radi of the clusters (size k) */
 		SGVector<float64_t> R;
 
+        /** Std of the clusters (size k) */
+        SGMatrix<float64_t> stds;
```

Comment:
I'd prefer if we didnt store this in the class but rather compute it on demand by a user call